test/functional/test_framework/test_framework.py

```diff
@@ -158,6 +161,21 @@ def main(self):
             self.options.tmpdir = tempfile.mkdtemp(prefix=TMPDIR_PREFIX)
         self._start_logging()
 
+        # Seed the PRNG.  Note that testruns are only reproducible if only a
+        # single thread accesses the PRNG (see
+        # https://docs.python.org/3/library/random.html#notes-on-reproducibility).
+        # The network thread shouldn't access random. If we need to change the
```

Comment:
Alternatively:
```python3
        # Seed the PRNG. Note that testruns are reproducible if and only if
        # a single thread accesses the PRNG. For more information, see
        # https://docs.python.org/3/library/random.html#notes-on-reproducibility.
```

Reply:
Thanks. That's better. I've updated the comment.